models/core/src/security.ts

```diff
@@ -35,11 +35,26 @@ export class TSpace extends TDoc implements Space {
 /**
  * @public
  */
-@Model(core.class.Account, core.class.Doc, DOMAIN_MODEL)
-export class TAccount extends TDoc implements Account {
+@Model(core.class.Person, core.class.Doc)
+export class TPerson extends TDoc implements Person {
   email!: string
-  name!: string
   firstName!: string
   lastName!: string
   avatar!: string
+  bio!: string
+  phone!: string
+  address!: {
+    country?: string
+    city?: string
+    street?: string
+    zip?: string
+  }
+}
+
+/**
+ * @public
+ */
+@Model(core.class.Account, core.class.Person, DOMAIN_MODEL)
+export class TAccount extends TPerson implements Account {
```

Comment:
I think better is to have `contact` plugin with all contact related stuff inside. And hold account only for some profile information. Person could have a link to account, or it could be matched by email if needed.